tests/excel/test_range_specification.py

```
@@ -0,0 +1,276 @@
+        # 逆転した範囲(C3:A1は無効)
+        with pytest.raises(
+            ValueError, match="Invalid range: end cell must be after start cell"
+        ):
+            self.loader.load_from_excel_with_range(excel_path, range_spec="C3:A1")
+
+    def test_directive_range_option(self):
+        """JsonTableDirectiveの:range:オプションテスト(未実装なので失敗する)。"""
+        excel_path = self.create_range_test_excel()
+
+        # モックSphinx環境
+        class MockConfig:
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)